### PR TITLE
CMake: use miniupnpc target instead of global vars

### PR DESCRIPTION
**Pull request by ligfx**

### CODE REVIEW

**Comment by ligfx** [comment]
@Orphis

File: CMake/FindMiniupnpc.cmake [hunk 1]
```diff
@@ -17,3 +17,11 @@ find_package_handle_standard_args(MINIUPNPC DEFAULT_MSG MINIUPNPC_INCLUDE_DIR MI
 set(MINIUPNPC_LIBRARIES ${MINIUPNPC_LIBRARY})
 set(MINIUPNPC_INCLUDE_DIRS ${MINIUPNPC_INCLUDE_DIR})
 mark_as_advanced(MINIUPNPC_INCLUDE_DIR MINIUPNPC_LIBRARY MINIUPNPC_API_VERSION_STR)
+
+if (MINIUPNPC_FOUND AND NOT TARGET miniupnpc)
+  add_library(miniupnpc UNKNOWN IMPORTED)
+  set_target_properties(miniupnpc PROPERTIES
```
Comment (Orphis) [inline CMake/FindMiniupnpc.cmake]: Targets in Find* modules are usually namespaced. Could you do that?
You would just need to add an alias in the concrete target if we build it in source.

Comment (ligfx) [inline CMake/FindMiniupnpc.cmake]: Sure. Where's a good place to create the alias? Where `find_package(Miniupnpc)` is called, or in miniupnpc's CMakeLists.txt?

Comment (Orphis) [inline CMake/FindMiniupnpc.cmake]: `add_library(Miniupnpc::miniupnpc UNKNOWN IMPORTED)`  in the Find module.
`add_library(Miniupnpc::miniupnpc ALIAS miniupnpc)` in its CMakeLists.txt

Comment (ligfx) [inline CMake/FindMiniupnpc.cmake]: Done.

**Pull request merged**